tasks/internal_reconcile_report_orphan/src/use_cases/get_orphans_page.py

```diff
@@ -1,13 +1,16 @@
-import logging
+from aws_lambda_powertools import Logger
 
 from src.use_cases.adapter_interfaces.storage import OrphansPageStorageInterface
 from src.entities.orphan import OrphanRecordPage, OrphanRecordFilter
 
+# Set AWS powertools logger
+Logger = Logger()
+
 
 def task(
         orphan_record_filter: OrphanRecordFilter,
         orphans_page_storage: OrphansPageStorageInterface,
-        logger: logging.Logger
+        logger: Logger
```

Comment:
You should not use an instantiated class as a typehint, and the typehint should still be logging.Logger. Revert changes to this file.

Reply:
reverted